storage/keyring_test.go

```diff
@@ -315,3 +380,11 @@ func (suite *KeyringSuite) TestUnmarshalFailure() {
 
 	storageDataUnmarshal = json.Unmarshal
 }
+
+func TestManageBytes(t *testing.T) {
+	data := []byte{65, 66, 67, 226, 130, 172}
+	splitted_data := [][]byte{{65, 66}, {67, 226, 130, 172}}
+
+	b := joinByteArray(splitted_data)
+	assert.Equal(t, b, data)
```

Comment:
the expected value is the 2nd arg, not the 3rd: https://pkg.go.dev/github.com/stretchr/testify/assert#Equal

I haven't looked at the tests above, so may apply above as well.

Reply:
🤦  got it.